src/OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth/Weather/Manager.cpp

```diff
@@ -0,0 +1,790 @@
+/// Apache License 2.0
+
+#include <chrono>
+#include <cstdlib>
+#include <fstream>
+#include <numeric>
+#include <thread>
+
+#include <OpenSpaceToolkit/Core/Containers/Map.hpp>
+#include <OpenSpaceToolkit/Core/Error.hpp>
+#include <OpenSpaceToolkit/Core/FileSystem/Path.hpp>
+#include <OpenSpaceToolkit/Core/Types/Integer.hpp>
+#include <OpenSpaceToolkit/Core/Types/String.hpp>
+#include <OpenSpaceToolkit/Core/Utilities.hpp>
+
+#include <OpenSpaceToolkit/IO/IP/TCP/HTTP/Client.hpp>
+
+#include <OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth/Weather/Manager.hpp>
+#include <OpenSpaceToolkit/Physics/Time/Date.hpp>
+#include <OpenSpaceToolkit/Physics/Time/DateTime.hpp>
+#include <OpenSpaceToolkit/Physics/Time/Instant.hpp>
+#include <OpenSpaceToolkit/Physics/Time/Scale.hpp>
+#include <OpenSpaceToolkit/Physics/Time/Time.hpp>
+
+#include <experimental/filesystem>
```

Comment:
What is this? O_o

Reply:
It's to check file size:
`std::experimental::filesystem::file_size`

but looks like it's no longer experimental! Changed to `#include <filesystem>`